docs/src/templates/pages/styleguide/ui-components/buttons.hbs

```diff
@@ -10,12 +10,15 @@ layout: components
 ---
 
 <h2 class="sg-sectionHeading">Default Button – <code>.o-btn</code></h2>
+<p>The default button class sets up the defaults for the other modifiers to build upon.  This is just used as a base and it should always have a second class associated with it to define the specific button type.</p>
+
 {{#>demo-block }}
 {{> je-btn-simple text='Button' classname='o-btn' }}
 {{/demo-block }}
 
 
 <h2 class="sg-sectionHeading">Button Background Color Modifiers</h2>
+<p>The primary button design.  This essentially the primary button style used on the website.</p>
```

Comment:
`This is essentially..` Looks like there's a double space here as well?

Reply:
Fixed